Mage/src/main/java/mage/cards/decks/importer/MtgaImporter.java

```diff
@@ -22,38 +22,52 @@ public class MtgaImporter extends PlainTextDeckImporter {
             "(\\p{Digit}+)" +
                     "\\p{javaWhitespace}+" +
                     "(" + CARD_NAME_PATTERN.pattern() + ")" +
-                    "\\p{javaWhitespace}+" +
-                    "\\((\\p{Alnum}+)\\)" +
-                    "\\p{javaWhitespace}+" +
-                    "(\\p{Digit}+)");
+                    "(?:\\p{javaWhitespace}+\\()?" +
+                    "(\\p{Alnum}+)?" +
+                    "(?:\\)\\p{javaWhitespace}+)?" +
+                    "(\\p{Graph}+)?");
 
     private final CardLookup lookup = getCardLookup();
     private boolean sideboard = false;
 
     @Override
     protected void readLine(String line, DeckCardLists deckList, FixedInfo fixedInfo) {
+        
+        line = line.trim();
+        
+        if (line.equals("Deck")) {
+            return;
+        }
 
-        if (line.trim().equals("")) {
+        if (line.equals("Sideboard") || line.equals("")) {
```

Comment:
Are the two assumptions here that?:
1. Sideboard always shows up afterwards.
2. The first empty line in the file is between the the last entry in the deck and sideboard.

Reply:
Hi, thanks for the code review! That's right, the mtga-format is formulaic in the way that the maindeck always comes first, then an empty line, and then the sideboard.

Technically since the sideboard section is labeled with "Sideboard", we could change the importer to just skip empty lines, but I wouldn't want to change the existing functionality because it's not necessary.